templates/pages/about.html.twig

```diff
@@ -37,13 +42,18 @@
     <p>{{ 'about.list_of_used_libraries'|trans }}{#  #}</p>
 
     <ul>
-        <li>The awesome <a href="https://symfony.com/4">Symfony 4, PHP framework</a> and a lot of the
-            <a href="http://symfony.com/">Symfony components</a>.</li>
-        <li>The <a href="http://www.doctrine-project.org">Doctrine DBAL</a> for database access and abstraction.</li>
-        <li><a href="http://twig.sensiolabs.org/">Twig, flexible, fast, and secure template engine</a>.</li>
-        <li><a href="http://twitter.github.com/bootstrap">Bootstrap 4</a> and
-            <a href="http://fortawesome.github.com/Font-Awesome/">Font Awesome</a></li>
-        <li><a href="http://ckeditor.com/">CKEditor</a> for the clean HTML5 Wysywig editor areas.</li>
+        <li>The awesome <a href="https://symfony.com/">Symfony 4, PHP framework</a> and a lot of the
+            <a href="https://symfony.com/">Symfony components</a>.</li>
+        <li>The <a href="https://www.doctrine-project.org">Doctrine ORM and DBAL</a> for database access, abstraction and object mapping.</li>

```

Comment:
Doctrine ORM and DBAL -> Doctrine

- because we also use Migrations, Fixtures, etc.